spot-contracts/contracts/PerpetualTranche.sol

```diff
@@ -470,12 +471,42 @@ contract PerpetualTranche is
     }
 
     /// @inheritdoc IPerpetualTranche
-    /// @dev CAUTION: Only the whitelisted vault can call this function.
+    /// @dev Only the whitelisted vault can call this function.
+    function claimFees(address to) external override onlyVault afterStateUpdate nonReentrant whenNotPaused {
+        IERC20Upgradeable perp_ = IERC20Upgradeable(address(this));
+        uint256 collectedBal = perp_.balanceOf(address(perp_));
+        if (collectedBal > 0) {
+            perp_.safeTransfer(to, collectedBal);
+        }
+    }
+
+    /// @inheritdoc IPerpetualTranche
+    /// @dev Only the whitelisted vault can call this function.
+    function payProtocolFee() external override onlyVault afterStateUpdate nonReentrant whenNotPaused {
+        uint256 protocolSharePerc = feePolicy.protocolSharePerc();
+        if (protocolSharePerc > 0) {
+            _mint(feePolicy.protocolFeeCollector(), protocolSharePerc.mulDiv(totalSupply(), ONE - protocolSharePerc));
+        }
+    }
+
+    /// @inheritdoc IPerpetualTranche
+    /// @dev Only the whitelisted vault can call this function.
     ///      The logic controlling the frequency and magnitude of debasement should be vetted.
+    ///      Pays a protocol fee on every rebalance.
```

Comment:
Nothing in this function calls a protocol fee, so I think this comment is better placed somewhere else.